runtime/core.go

```diff
@@ -1181,6 +1227,14 @@ func hashNotImplemented(f *Frame, o *Object) (*Object, *BaseException) {
 	return nil, f.RaiseType(TypeErrorType, fmt.Sprintf("unhashable type: '%s'", o.typ.Name()))
 }
 
+func hexNotImplemented(f *Frame, o *Object) (*Object, *BaseException) {
```

Comment:
Prefer to do this raise inline rather than as a helper. Use helpers only when there is a significant amount of code being factored out or there is a lot of duplication.